README

```diff
@@ -117,8 +117,10 @@ for more information about either the memkind or the jemalloc configuration opti
 ## jemalloc
 The jemalloc source was forked from jemalloc version 5.2.1.  This source tree
 is located within the jemalloc subdirectory of the memkind source.  The jemalloc
-source code has been kept close to the original form, and in particular
-the build system has been lightly modified.
+source code has been kept close to the original form, with exceptions:
+- extend jemalloc API with "arenalookupx" function
+- optimization for looking correct command in mallctl
```

Comment:
optimization for searching commands in mallctl

Reply:
Done.